src/AzureExtension/Assets/QuickStartPlayground/Samples/rust-linked-list/.devcontainer/Dockerfile

```diff
@@ -3,5 +3,5 @@ FROM mcr.microsoft.com/devcontainers/base:ubuntu
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt update && \
-apt install -y curl && build-essentials \
+apt install -y curl && build-essential \
```

Comment:
I noticed this problem in the Rust sample (there's no `build-essentials` package), so I fixed that while I was in here.